README.md

```diff
@@ -17,6 +17,7 @@ These are currently:
 * SSLLabs
 * webbkoll
 * webhint
+* yellowlab
```

Comment:
```suggestion
* Yellow Lab Tools
```

Reply:
I complted your suggestion in an amended commit.